src/test/java/jenkins/plugins/openstack/compute/JCloudsCloudTest.java

```diff
@@ -276,6 +277,7 @@ public void globalConfigMigrationFromV1() throws Exception {
         assertEquals("/tmp/jenkins", to.getFsRoot());
         assertEquals("jenkins-testing", to.getKeyPairName());
         assertThat(to.getLauncherFactory(), instanceOf(LauncherFactory.SSH.class));
+        assertThat(to.getNodeProperties(), empty());
```

Comment:
This will be `null` after the suggested change.

Reply:
Done in https://github.com/jenkinsci/openstack-cloud-plugin/pull/270/commits/5456d1d835b8da5588b9f7e3ce96730cc0b1a071